popup/search-results.js

```diff
@@ -578,6 +579,19 @@ window.addEventListener('showStyles:done', function _() {
           return result.style_settings;
         });
     }
+
+    function fetchMd5(result) {
+      return API.download({
+        url: UPDATE_URL.replace('%', result.id),
+        timeout: 60e3,
```

Comment:
Why passing these options? We only send the URL to download API in update.js:
https://github.com/openstyles/stylus/blob/3d3a2b802aaf00fc0d728822793dfd163b76f9b0/background/update.js#L150